app/uk/gov/hmrc/cgtpropertydisposals/actors/TaxEnrolmentRetryManager.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.actors
+
+import akka.actor.{Actor, ActorRef, ActorRefFactory, Props, Scheduler, Terminated}
+import com.typesafe.config.{Config, ConfigFactory}
+import net.ceedubs.ficus.Ficus._
+import uk.gov.hmrc.cgtpropertydisposals.actors.TaxEnrolmentRetryConnectorActor.RetryCallToTaxEnrolmentService
+import uk.gov.hmrc.cgtpropertydisposals.actors.TaxEnrolmentRetryManager.{MarkTaxEnrolmentRetryRequestAsSuccess, RetryTaxEnrolmentRequest}
+import uk.gov.hmrc.cgtpropertydisposals.actors.TaxEnrolmentRetryMongoActor.{DeleteTaxEnrolmentRetryRequest, GetAllNonFailedTaxEnrolmentRetryRequests, UpdateTaxEnrolmentRetryStatusToFail}
+import uk.gov.hmrc.cgtpropertydisposals.models.TaxEnrolmentRequest
+import uk.gov.hmrc.cgtpropertydisposals.repositories.TaxEnrolmentRetryRepository
+import uk.gov.hmrc.cgtpropertydisposals.service.TaxEnrolmentService
+import uk.gov.hmrc.cgtpropertydisposals.util.Logging
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.duration.{FiniteDuration, _}
+
+class TaxEnrolmentRetryManager(
+  createMongoActor: ActorRefFactory => ActorRef,
+  createConnectorActor: ActorRefFactory => ActorRef,
+  taxEnrolmentRetryRepository: TaxEnrolmentRetryRepository,
+  taxEnrolmentService: TaxEnrolmentService,
+  scheduler: Scheduler
+) extends Actor
+    with Logging {
+  import context.dispatcher
+  implicit val hc: HeaderCarrier = HeaderCarrier()
+
+  val config: Config = ConfigFactory.load()
```

Comment:
you can a `Config` from the system somewhere - something like `system.configuration`, but I can't remember exactly